scss/_variables.scss

```diff
@@ -649,6 +649,14 @@ $box-shadow-inset:  null !default; // Boosted mod: instead of `inset 0 1px 2px r
 $component-active-color:      $black !default;
 $component-active-bg:         $primary !default;
 
+// scss-docs-start focus-ring-variables
+$focus-ring-width:      .25rem !default;
+$focus-ring-opacity:    .25 !default;
+$focus-ring-color:      rgba($primary, $focus-ring-opacity) !default;
+$focus-ring-blur:       0 !default;
```

Comment:
To remove on our side ?

Reply:
Removed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1838/commits/2610fc5a87bcf2bf962d94a969a088f13a2d21b6.